umbraco/Umbraco.CMS.Backoffice
# Bugfix: focus directive refactor to support late case

Sometimes the umb-focus directive does not succeed in setting focus.
I dont know why after testing many scenarios and cases, everything seems fine, but it does not succeed. So to make it succeed I have refactored to it keeps trying as long as its the desired element for focus.

## Types of changes




## Motivation and context



## How to test?

- go to the trashbin
- then create a new document
- see that the name input gets focus.

## Screenshots (if appropriate)

## Checklist
